circus-train-core/src/main/java/com/hotels/bdp/circustrain/context/CommonBeans.java

```diff
@@ -124,53 +112,62 @@ private void putConfigurationProperties(Map<String, String> configurationPropert
     }
   }
 
-  @Bean
-  MetaStoreClientFactoryManager metaStoreClientFactoryManager(List<MetaStoreClientFactory> metaStoreClientFactories) {
-    return new MetaStoreClientFactoryManager(metaStoreClientFactories);
```

Comment:
We relied on Spring to inject a list before (You now hardcode the ThriftHiveMetaStoreClientFactory), we never used any of this? No plugins relying on this?